org.eclipse.languageserver/src/org/eclipse/languageserver/languages/InitializeLaunchConfigurations.java

```diff
@@ -62,7 +62,7 @@ public void earlyStartup() {
 				workingCopy.setAttribute(IExternalToolConstants.ATTR_BUILD_SCOPE, "${none}");
 				workingCopy.setAttribute(DebugPlugin.ATTR_CAPTURE_OUTPUT, true);
 				workingCopy.setAttribute(IExternalToolConstants.ATTR_LOCATION, "/usr/bin/node");
-				workingCopy.setAttribute(IExternalToolConstants.ATTR_TOOL_ARGUMENTS, "/home/mistria/git/omnisharp-node-client/languageserver/server.js");
+				workingCopy.setAttribute(IExternalToolConstants.ATTR_TOOL_ARGUMENTS, "/home/mistria/git/omnisharp-node-client/languageserver/server.js --stdio");
```

Comment:
Sorry, but this is against the principle of a review. I'd rather avoid as much as possible polluting the Git repository with erroneous commits.